.github/workflows/ci.yml

```diff
@@ -85,8 +92,20 @@ jobs:
           name: binaries_dev_${{ matrix.library }}
           path: binaries/
 
+  fail-if-target-branch:
+    name: Fail if target branch is specified
+    needs:
+      - get_dev_artifacts
+    if: needs.get_dev_artifacts.outputs.target-branch != ''
```

Comment:
Did you tested ? I don;t understand where the output of the `get_dev_artifacts` job is set 🤔